### DISCUSSION TITLE
Pivot Tables

### DISCUSSION THREAD

**Discussion by parko65** [Megabit/Blazorise#2143] [General]
Dear Blazorise Team,
I'm contemplating using Blazorise in an upcoming project and wondered if there is a template for creating Pivot Tables?

**Comment by stsrki** [comment]
It is not supported by the Blazorise. At this time we don't have the plans to include it. Maybe sometime later in the future.

The only option for you is to build it yourself using our Table and its sub-components. But it is not an easy task, I know...
